src/main/java/se/michaelthelin/spotify/model_objects/special/PlaybackQueue.java

```diff
@@ -123,17 +124,7 @@ public PlaybackQueue createModelObject(JsonObject jsonObject) {
     }
 
     private IPlaylistItem asPlaylistItem(JsonObject trackObj) {
-      IPlaylistItem item = null;
-      if (hasAndNotNull(trackObj, "type")) {
-        String type = trackObj.get("type").getAsString().toLowerCase();
-
-        if (type.equals("track")) {
-          item = new Track.JsonUtil().createModelObject(trackObj);
-        } else if (type.equals("episode")) {
-          item = new Episode.JsonUtil().createModelObject(trackObj);
-        }
-      }
-      return item;
+        return PlaylistItemFactory.createPlaylistItem(trackObj);
```

Comment:
The indentation looks a bit off to me. The `return` statement should only be indented using two spaces instead of four as defined in the `.editorconfig` file in the repository's root.

Reply:
Yes, you're right! Thanks